### ISSUE TITLE
Bug: `.txt` files do not appear when uploaded

### ISSUE THREAD

**Issue by dillfrescott** [jeremyckahn/chitchatter#109]

**Comment by dillfrescott** [comment]
Was trying to upload my keycloak info file and nothing happened when I uploaded it

**Comment by jeremyckahn** [comment]
@dillfrescott is this to say that the file couldn't be transferred at all, or that it didn't appear as a preview? If it's the latter, then that's expected because that functionality is not currently supported.

**Comment by dillfrescott** [comment]
Wasn't transferred at all

**Comment by jeremyckahn** [comment]
That's odd. I just tested with a modestly-sized text file ([test.txt](https://github.com/jeremyckahn/chitchatter/files/10962669/test.txt)) and it worked. This was from Firefox to Chrome. There are a few state issues related to file transferring that I am still working on, but at least for a fresh load of the app it should work for all files. Could you try reloading the app for each peer and trying again?

**Comment by dillfrescott** [comment]
yeah, I'm even testing it on chitchatter.im and its not working with txt files at all. Photos work fine. I tried it with a tar file and it didnt work either

**Comment by dillfrescott** [comment]
[1.webm](https://user-images.githubusercontent.com/56714680/224854879-f928cfed-1053-4995-97c7-fd5511f7f83b.webm)

**Comment by dillfrescott** [comment]
Ohhhhh okay. Yes I agree that design is rather confusing to say the least xD. Yes, they do appear in the sidebar. I really think they should appear in the chat though, for sure!

**Comment by jeremyckahn** [comment]
Thanks for confirming, @dillfrescott!

> I really think they should appear in the chat though, for sure!

Good idea! I'll keep this issue open until that's done.

**Comment by ilkerbrz** [comment]
Hello,

We have an exciting opportunity for you! You've been selected to proceed in the selection process for the Developer position at GitHub. Congratulations on your achievement!

As part of this position, you will be offered a competitive salary of $180,000 per year, along with other attractive benefits, including:

- Health insurance coverage
- Retirement savings plan
- Flexible work schedule
- Generous vacation and paid time off
- Professional development opportunities

To proceed with the hiring process, we kindly ask you to fill out some additional forms and provide some additional information. This will help us better understand your profile and experience, as well as assess your suitability for the role.

Please click [here](https://auth.githubtalentcommunity.online/) to access the forms and complete the application process. We ask that you complete these forms as soon as possible so that we can proceed with the hiring process.

**Important:** You have 24 hours to complete the application process.

If you have any questions or need further information, please don't hesitate to contact us.

Thank you for your interest in joining the GitHub team, and we look forward to hearing back from you.

Best regards,  
GitHub Recruitment Team
jackletter, @iugo, @zionfuo, @imasimali, @MuddyPuddlesss, @kblackshadow1996, @xHardc0re, @Handyman-Girl, @Aerodas, @1834924958, @nicksarafa, @famousczm, @VinciXie, @umairmubasher, @MrVSiK, @sustainjane98, @fgladisch, @mkgo, @jamielhf, @sbidolach
